engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -1191,6 +1191,14 @@ public void testInThePresenceOfComments() throws Exception {
     """);
   }
 
+
+  @Test
+  public void testNPE183892665() throws Exception {
+    parseTest("""
+    foo : Integer ->
+    """);
+  }
```

Comment:
What is the parser result now?

Is there no syntax error there?

Reply:
There is an `IR.Syntax.Error` in both - the old and new parser.

Reply:
Thanks for the explanation